cpp/src/io/csv/csv_reader.cu

```diff
@@ -984,7 +989,7 @@ gdf_error launch_dataConvertColumns(raw_csv_t *raw_csv, void **gdf, gdf_valid_ty
 		num_valid
 	);
 
-	CUDA_TRY( cudaGetLastError() );
+	CUDA_TRY( cudaGetLastError());
```

Comment:
Since the previous PR was accepted, merge conflicts resulted in having to manually copy the previous changes back, hence the unintentional extra spaces. Will proceed to fix these trivial things.